### PR TITLE
S2t2

### PR DESCRIPTION
**Pull request by Phildallen**

### CODE REVIEW

File: src/components/DetailModal/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }, [])
+
+
+    return (
+        <>
+            <div class="container">
+            <div class="row d-flex">
+            <p class="fs-6"><strong>{data.job_title} - {data.company}</strong></p>
+            <button type="button" class="btn-close" disabled aria-label="Close"></button>
+        </div>
```
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Indentation

File: src/components/DetailModal/index.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            </div>
+            </div>
+        </>
+    )
+}
+
+export default DetailModal;
+
+
+
```
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: too many blank end spaces

File: src/components/DetailModal/index.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                                        <td scope="col">Type:</td>
+                                        <td scope="col">Date posted:</td>
+                                        <td scope="col">Skills:</td>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    <tr>
+                                        <td>£{data.salary ?? "Not specified"}</td>
+                                        <td>{data.type ?? "Not specified"}</td>
+                                        <td>{data.posted}</td>
```
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Posted needs to be converted from International date format into UK date

File: src/components/DetailModal/index.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+                                        <td>
+                                            <div className="d-flex align-items-center column-gap-4 fs-6"><strong>
+                                                {
+                                                    skills && skills.map((element, i) =>
+                                                        <span className="rounded-pill bg-info text-white py-0 px-1"
```
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Buttons look a bit too round compared to the brief. Could they be more rectangular?

File: src/components/DetailModal/index.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from 'react';
```
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: semi colons at end of lines - consistency - haven't always used them

**Review by JamesMcGraaw — CHANGES_REQUESTED** [review]
Looks good, a few more changes need to be made before approval

File: src/components/DetailModal/index.js [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+                                    <strong>{data.job_title} - {data.company}</strong></p>
+                                <button type="button" className="btn-close" data-bs-dismiss="modal"
+                                        aria-label="Close"></button>
+                            </div>
+                            <div className="modal-body">
+                                <div className="container">
+
+                                    <div className="d-flex flex-row bg-primary text-white py-5 fs-4">
+                                        <p className="mx-4"><strong>{data.job_title}</strong></p>
+                                        <p className="bg-success text-white rounded-pill px-3">
```
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Needs to go under title on mobile

File: src/components/DetailModal/index.js [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+                                            <strong>{data.type}</strong>
+                                        </p>
+                                    </div>
+
+                                    <div class="border border-5 border-light p-3">
+
+                                        <div className="row mx-auto" pt-3>
+                                            <img src={data.logo} className="col-2"/>
```
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js]: Gets warped shape on mobile view

File: src/components/DetailModal/index.js [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+                                        <table class="table table-borderless table-sm">
+
+                                            <thead>
+                                            <tr>
+                                                <td scope="col">Salary:</td>
+                                                <td scope="col">Type:</td>
+                                                <td scope="col">Date posted:</td>
+                                                <td scope="col">Skills:</td>
+                                            </tr>
```
Comment (JamesMcGraaw) [inline src/components/DetailModal/index.js:66]: this text could all be a little smaller, especially looking on mobile

File: src/components/DetailModal/index.js (hunk 3 above)
Comment (devhenlee) [reply src/components/DetailModal/index.js]: Done!

File: src/components/DetailModal/index.js (hunk 4 above)
Comment (devhenlee) [reply src/components/DetailModal/index.js]: Done!

File: src/components/DetailModal/index.js (hunk 5 above)
Comment (devhenlee) [reply src/components/DetailModal/index.js]: Done!

File: src/components/DetailModal/index.js (hunk 6 above)
Comment (devhenlee) [reply src/components/DetailModal/index.js]: Done!

**Pull request closed without merging**